ENG-622 - 1.60.0 - OIDC prompt parameter (#3891)
FusionAuth/fusionauth-site: astro/src/content/docs/lifecycle/authenticate-users/single-sign-on.mdx

## Configure The Applications In FusionAuth

> Next, create and configure the applications in FusionAuth. You can do this via the API, but in this guide it'll be accomplished through the administrative user interface.

Reviewer: ```suggestion
Next, create and configure the applications in FusionAuth. This guide uses the Admin UI, though you can also do this using the API.
```
